core/src/main/java/io/questdb/griffin/engine/table/AsyncGroupByAtom.java

```diff
@@ -365,6 +396,17 @@ public void tryShard(Particle particle) {
         }
     }
 
+    private void prepareBindVarMemory(SymbolTableSource symbolTableSource, SqlExecutionContext executionContext) throws SqlException {
+        // don't trigger memory allocation if there are no variables
```

Comment:
nit: this looks like a duplicate code

Reply:
Removed code duplication in 66a07d5bdf4d63500f211077e72e8c71fa29f4af